ceph.spec.in

```diff
@@ -153,6 +156,15 @@ BuildRequires:  openldap-devel
 BuildRequires:  openssl-devel
 BuildRequires:  redhat-lsb-core
 BuildRequires:	Cython
+%if 0%{?rhel}
+BuildRequires:	python34-devel
+BuildRequires:	python34-setuptools
+BuildRequires:	python34-Cython
+%else
+BuildRequires:	python3-devel
+BuildRequires:	python3-setuptools
+BuildRequires:	python3-Cython
+%endif
 %endif
```

Comment:
yes, after moving the conditional block and deleting the duplicate conditionals from the suse conditional block, the comment is only thing that is new.